app.py

```diff
@@ -21,8 +21,8 @@ def predict():
     #predict = prediction[:, 1]

```

Comment:
GPT summary of [53fcbf](https://github.com/SampatSuthar/ml-demo/blob/2ce9a211365f18cfb46299b15c0e532c3e8af178/app.py#53fcbfb6eff29c32752a36cbdcba9f7463255086) - [061389](https://github.com/SampatSuthar/ml-demo/blob/eff3c18ac2d2a62721066bb2d7b050cd14c87962/app.py#061389ba9074b67386bd3c72ead2de702f8e9e3d):
* Removed the import of the openai module
* Commented out the line to set the openai API key
* Added a prompt and model for the openai completion